src/js/components/SearchPage.js

```diff
@@ -11,6 +11,36 @@ import TabBar from "./TabBar/TabBar"
 import TabContent from "./TabContent"
 import Tabs from "../state/Tabs"
 import useSearchShortcuts from "./useSearchShortcuts"
+import styled from "styled-components"
+
+const ColumnLayout: React$ComponentType<any> = styled.div`
```

Comment:
I hit some flow errors here which baffled me, and adding this type annotation/assertion was the only solution that worked. Happy to take different suggestions!